themes/swift-hep/layouts/index.html

```diff
@@ -10,6 +10,7 @@ <h2 class="text-secondary h3">{{ . }}</h2>
             </div>
         </div>
     </div>
+-->
```

Comment:
If you are happy with how this looks, you can just remove the code-block instead of commenting it out